### PR TITLE
Removed superfluous details from summary response object

### PR DESCRIPTION
**Pull request by kunagpal**
https://github.com/postmanlabs/newman/pull/772#discussion_r89292749

### CODE REVIEW

**Comment by kunagpal** [comment]
@czardoz That was the initial idea, but the `response.size` and `response.json` methods are used within the run instance, so I can't directly use the plain JSONified SDK object.

**Comment by czardoz** [comment]
@kunagpal 
There's something specific about the response object that needs fixing. If this was a generic problem, we would have seen it in the `request` too.

**Comment by kunagpal** [comment]
@czardoz Fixed.

File: lib/reporters/html/index.js [hunk 1]
```diff
@@ -63,6 +63,8 @@ PostmanHTMLReporter = function (newman, options) {
                     // set base values for overall response size and time values
                     _.set(executionMeans, executionId, { time: { sum: 0, count: 0 }, size: { sum: 0, count: 0 } });
 
+                    currentExecution.response = currentExecution.response.toJSON();
```
Comment (czardoz) [inline lib/reporters/html/index.js]: This mutates the executions array in `summary.run.executions`, and therefore is not an acceptable solution.

**Pull request merged**